tools/mo/openvino/tools/mo/middle/SSliceComplex.py

```diff
@@ -109,7 +109,8 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
 
         for dst in complex_node.out_port(0).get_connection().get_destinations():
             after_complex_node = dst.node
-            after_complex_node['input_rank_changed'] = True
+            if after_complex_node.type == 'Roll':
+                add_constant_to_negative_values(after_complex_node, 2, int64_array(emulated_complex_tensor_rank))
```

Comment:
Mateusz, now you actually adjust `Roll` axis. I think we no longer need `CorrectRollAxes` front transformation then. Please remove it.

Reply:
done